fpmsyncd/fpmlink.cpp

```diff
@@ -276,6 +276,13 @@ void FpmLink::processFpmMessage(fpm_msg_hdr_t* hdr)
             /* EVPN Type5 Add route processing */
             processRawMsg(nl_hdr);
         }
+#ifdef HAVE_NEXTHOP_GROUP
```

Comment:
Why do we need this macro check? Can we do a dynamic check like

DEVICE_METADATA['localhost']['nexthop_group'] 

This is done as part of PR - https://github.com/sonic-net/sonic-buildimage/pull/16762/files

Could you pls check this?